package.json

```diff
@@ -9,6 +9,7 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@gadget-client/doc-inspector": "^1.382.0",
     "@gadget-client/my-ai-app-mdw": "^1.34.0",
```

Comment:
Looks good! Is there any need to keep the api for `my-ai-app-mdw`?

Reply:
Ah I just saw this. For some reason this comment didn't appear on mobile as expected 😓 . Maybe we'll remove it in another commit.

Reply:
No worries! I removed it and made a commit before merging